magento/pwa-studio
# Zetlen/inline logo svg

## Description
Fixes #1622. Also Magento treats the path "/static" as special, so changed to "/venia-static"

## Related Issue





Closes #1622 

## Verification Steps

1. Verify #1622 works fine. 
2. For upward-php and js make sure no network call ends up in errors like 404, 500 etc.
